src/Compilers/Server/VBCSCompilerTests/CompilerServerTests.cs

```diff
@@ -38,6 +38,7 @@ static CompilerServerUnitTests()
                 !File.Exists(Path.Combine(basePath, CSharpClientExeName)) ||
                 !File.Exists(Path.Combine(basePath, BasicClientExeName)))
             {
+#if false
```

Comment:
I'm not sure what this is for, and so how to deal with the call to `DesktopTestHelpers.GetMSBuildDirectory` here.

Reply:
just `#if` the entire static ctor for desktop only